eng/common/scripts/Helpers/Package-Helpers.ps1

```diff
@@ -1,37 +1,47 @@
 function GetPackageKey($pkg) {
-    $pkgKey = $pkg.Package
-    $groupId = $null
-  
-    if ($pkg.PSObject.Members.Name -contains "GroupId") {
-      $groupId = $pkg.GroupId
-    }
-  
-    if ($groupId) {
-      $pkgKey = "${groupId}:${pkgKey}"
-    }
-  
-    return $pkgKey
+  $pkgKey = $pkg.Package
+  $groupId = $null
+
+  if ($pkg.PSObject.Members.Name -contains "GroupId") {
+    $groupId = $pkg.GroupId
   }
+
+  if ($groupId) {
+    $pkgKey = "${groupId}:${pkgKey}"
+  }
+
+  return $pkgKey
+}
   
-  # Different language needs a different way to index the package. Build a map in convienice to lookup the package.
-  # E.g. <groupId>:<packageName> is the package key in java.
-  function GetPackageLookup($packageList) {
-    $packageLookup = @{}
-  
-    foreach ($pkg in $packageList) {
-      $pkgKey = GetPackageKey $pkg
-  
-      # We want to prefer updating non-hidden packages but if there is only
-      # a hidden entry then we will return that
-      if (!$packageLookup.ContainsKey($pkgKey) -or $packageLookup[$pkgKey].Hide -eq "true") {
-        $packageLookup[$pkgKey] = $pkg
-      }
-      else {
-        # Warn if there are more then one non-hidden package
-        if ($pkg.Hide -ne "true") {
-          Write-Host "Found more than one package entry for $($pkg.Package) selecting the first non-hidden one."
-        }
+# Different language needs a different way to index the package. Build a map in convienice to lookup the package.
+# E.g. <groupId>:<packageName> is the package key in java.
+function GetPackageLookup($packageList) {
+  $packageLookup = @{}
+
+  foreach ($pkg in $packageList) {
+    $pkgKey = GetPackageKey $pkg
+
+    # We want to prefer updating non-hidden packages but if there is only
+    # a hidden entry then we will return that
+    if (!$packageLookup.ContainsKey($pkgKey) -or $packageLookup[$pkgKey].Hide -eq "true") {
+      $packageLookup[$pkgKey] = $pkg
+    }
+    else {
+      # Warn if there are more then one non-hidden package
+      if ($pkg.Hide -ne "true") {
+        Write-Host "Found more than one package entry for $($pkg.Package) selecting the first non-hidden one."
       }
     }
-    return $packageLookup
   }
+  return $packageLookup
+}
+
+# For deprecated packages, add "(deprecated)" besides of display name.
+function GetDocsTocDisplayName($pkg) {
```

Comment:
Where is this getting used?

Reply:
This is needed for language-setting in each language repo

Reply:
I have a PR for your reference. https://github.com/Azure/azure-sdk-for-net/pull/33109

Triggered the pipeline: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=2065510&view=results